documents/submitting-a-pr-to-hoodie.md

```diff
@@ -0,0 +1,27 @@
+So, you’ve written a blog post and you’d like to submit it to the [Hood.ie Blog](http://hood.ie/blog)? Firstly, a :star: big thank you :star: from all of us, we really appreciate your work. Keep reading to find out which path on the 'submitting a blog post' you need to take on the journey you’re about to embark on :earth_africa:.
+
+## It’s dangerous to go alone, take this:
+
+You are:
+* [Confident with submitting pull requests](#i-am-confident-in-submitting-pull-requests)
+* [Not-so-confident but like to do the submission on the command line]
+* [Not-so-confident but want to do the submission in the browser]
+
+## I am confident in submitting pull-requests!
+
+We won’t take up much of your time then and will give you the bullet-pointed version:
+
+* Make sure you have [ruby](https://www.ruby-lang.org/en/) installed on your computer, as well as the the [`jekyll`](https://jekyllrb.com/) rubygem :gem:.
+* Fork and clone the [Hood.ie website](https://github.com/hoodiehq/hood.ie) repository.
+* Put your blog post (in markdown format) into the `_drafts` folder of the repository, following the correct post format depending on what your post is (TGIF, a regular post, something else). [Formats can be found here](https://github.com/hoodiehq/hood.ie/tree/gh-pages/_drafts).
+* Ensure it looks how you want by running the `jekyll serve --watch --drafts` command on the command line, making any edits you want.
+* When satisfied with your post, rename the file for the post you want to publish to YEAR-MONTH-DAY-your-post-title.md (e.g. 2015-11-26-hoodie-editorial-team.md) and move it from the `_drafts` folder to the `_posts` folder.
+* Commit your blog post and push the branch up.
```

Comment:
maybe include something to ensure that the branch to make the pull request to should be gh-pages... caused me a bit of a bother :relaxed: